packages/calling/calling/src/CallingClient/registration/register.ts

```diff
@@ -672,11 +646,11 @@ export class Registration implements IRegistration {
           body,
           (clientError, finalError) => {
             if (finalError) {
-              this.emitter(EVENT_KEYS.ERROR, undefined, clientError);
+              this.callingClientEmitter(EVENT_KEYS.ERROR, clientError);
```

Comment:
We should emit line error here, applicable at other places also in this file. From this file all the should be of type line error

Reply:
`handleRegistrationErrors` is called from both `register.ts` and `callingClient.ts`. We can't assign `LineErrorEmitterCallback` and `CallingClientErrorEmitterCallback` to handleRegistrationErrors emitterCb parameter. We can discuss more on this.

Reply:
As discussed, this will be taken care in next story as it requires significant amount of changes